tests/storages_tests/rdb_tests/test_with_server.py

```diff
@@ -35,6 +37,23 @@ def run_optimize(args: Tuple[str, str]) -> None:
     study.optimize(objective, n_trials=20)
 
 
+def run_optimize_with_progbar(args: Tuple[str, str, int]) -> str:
+    study_name = args[0]
+    storage_url = args[1]
+    n_jobs = args[2]
+    # Create a study
+    study = optuna.load_study(study_name=study_name, storage=storage_url)
+
+    tmp = tempfile.NamedTemporaryFile()
+    with open(tmp.name, "w") as f:
+        with contextlib.redirect_stderr(f):
+            # Run optimization
```

Comment:
This sounds obvious:
```suggestion
```

Reply:
I added this comment because existing ```run_optimize``` has same comment before ```study.optimize```.

Reply:
The existing code might not be a ground truth; personally, `run_optimize`'s comment looks also unnecessary.

Reply:
Ok, removed.